src/linter/tests/fixtures/invalidChangeVersion-environment.mjs

```diff
@@ -7,7 +7,7 @@ const issues = invalidChangeVersion([
     ...assertEntry,
     changes: [
       ...assertEntry.changes,
-      { version: ['SOME_OTHER_RELEASED_VERSION'] },
+      { version: ['SOME_OTHER_RELEASED_VERSION', 'v0.1.2'] },
```

Comment:
maybe add comment to explain why this version is hardcoded

Reply:
@aduh95 I'm assuming the reason is they are not included in the changelog, just confirming with you?